app/models/item.rb

```diff
@@ -1,3 +1,7 @@
 class Item < ApplicationRecord
   belongs_to :supermarket
-end
\ No newline at end of file
+  has_many :customers
```

Comment:
we want to have similar associations here as the customer model (but reverse -- has_many :customers, through: :customer_items)